task-2/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
```

Comment:
Let's add 6 additional script commands:
`"prettier": "....",`
`"prettier:fix": "....",`
`"eslint": "....",`
`"eslint:fix": "....",`
`"lint": "npm run prettier && npm run eslint",`
`"lint:fix": "npm run prettier:fix && npm run eslint:fix",`

The exact details of the first 4 commands is up to you to figure out.
Hint: `*:fix` command not only checks for errors, but also auto-fixes them where possible.

Reply:
All 4 relevant scripts have been added.